stylesheet/components/cart.css

```diff
@@ -0,0 +1,53 @@
+.cart-container{
+     margin:  0 auto;
+}
+
+table{
+     width: 100%;
+     border-collapse: collapse;
+}
+
+.product-detail{
+     display: flex;
+     flex-wrap: wrap;
+}
+th{
+     text-align: left;
+     padding: 30px;
+     color: white;
+     background-color: var(--primary);
+     font-weight: normal;
+}
+
+td{
+     padding: 10px 5px;
+}
+
+td input{
+     width: 40px;
+     height: 30px;
```

Comment:
try to use rem unit

Reply:
here i have used px everywhere, next time i ll try to keep in rem.